conversions.tmpl

```diff
@@ -13,7 +13,7 @@ import (
 	"unsafe"
 )
 
-// Ptr takes a pointer, slice, or array and returns its GL-compatible address.
+// Ptr takes a pointer or slice and returns its GL-compatible address.
```

Comment:
Can we elaborate on "pointer", pointer to what is okay? Pointer to slice? Pointer to array? Pointer to first element of slice? Pointer to first element of array?

As far as I can tell, only pointers to first elements of slices/arrays are allowed. If so, that should be clearly documented here.